lib/Pod/To/Text.pm6

```diff
@@ -20,7 +20,25 @@ sub pod2text($pod) is export {
         when Pod::Block::Declarator { declarator2text($pod)     }
         when Pod::Item         { item2text($pod).indent(2)      }
         when Pod::FormattingCode { formatting2text($pod)        }
-        when Positional        { $pod.map({pod2text($_)}).join("\n\n")}
+        when Positional        {
+            =begin comment
+	    # original code
+            $pod.map({pod2text($_)}).join("\n\n")
+            =end comment
```

Comment:
We already have git history if ever need previous code. We don't need to keep it in source.